src/App.vue

```diff
@@ -50,36 +50,38 @@
               v-on:css-errors="cssErrors = $event"/>
           </b-col>
 
-          <b-col cols="4" class="overflow-hidden h-100 pr-0 pl-4">
-            <b-card no-body class="p-0">
+          <b-col class="overflow-hidden h-100 preview-inspector p-0">
+            <b-card no-body class="p-0 h-100">
               <b-card-header class="stick-top">
                 Inspector
               </b-card-header>
 
-              <b-card-body class="p-0">
-                <b-button v-b-toggle.dataInput variant="outline"
-                  class="text-left card-header d-flex align-items-center w-100"
-                  @click="showDataInput = !showDataInput">
-                  <i class="fas fa-file-import mr-2"></i>
-                    {{ $t('Data Input') }}
-                  <i class="fas fa-angle-down ml-auto" :class="{ 'fas fa-angle-right' : !showDataInput }"></i>
-                </b-button>
-
-                <b-collapse id="dataInput" visible class="overflow-auto">
-                  <form-text-area class="data-height h-100 dataInput"  v-model="previewInput"></form-text-area>
-                </b-collapse>
-
-                <b-button v-b-toggle.dataPreview variant="outline"
-                  class="text-left card-header d-flex align-items-center w-100"
-                  @click="showDataPreview = !showDataPreview">
-                  <i class="fas fa-file-code mr-2"></i>
-                    {{ $t('Data Preview') }}
-                  <i class="fas fa-angle-down ml-auto" :class="{ 'fas fa-angle-right' : !showDataPreview }"></i>
-                </b-button>
-
-                <b-collapse id="dataPreview" visible class="mt-2 overflow-auto">
-                  <vue-json-pretty  :data="previewData" class="p-2 data-height"></vue-json-pretty>
-                </b-collapse>
+              <b-card-body class="p-0 overflow-auto">
+                  <b-button variant="outline"
+                    class="text-left card-header d-flex align-items-center w-100 shadow-none"
+                    @click="showDataInput = !showDataInput">
+                    <i class="fas fa-file-import mr-2"></i>
+                      {{ $t('Data Input') }}
+                    <i class="fas ml-auto" :class="showDataInput ? 'fa-angle-right' : 'fa-angle-down'"></i>
+                  </b-button>
+
+                  <b-collapse v-model="showDataInput" id="showDataInput">
+                    <form-text-area class="data-input mb-0 data-collapse" v-model="previewInput"></form-text-area>
```

Comment:
Is seems this is required because of the margin on the label for the rendered `form-text-area`. You can clean this up (remove `data-input` and replace it with `h-100`) if you set `display: none` on the label inside this component.